compiler/src/dmd/location.d

```diff
@@ -0,0 +1,136 @@
+/**
+ * Encapsulates file/line/column locations.
+ *
+ * Copyright:   Copyright (C) 1999-2022 by The D Language Foundation, All Rights Reserved
+ * Authors:     $(LINK2 https://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 https://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/globals.d, _location.d)
+ * Documentation:  https://dlang.org/phobos/dmd_location.html
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/location.d
+ */
+
+module dmd.location;
+
+import dmd.common.outbuffer;
+import dmd.root.filename;
+import dmd.globals;
```

Comment:
Why the circular import ?

Reply:
Because I haven't yet completely disentangled location from globals, to keep the PR simple.